.changeset/many-taxis-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@openfun/cunningham-tokens": patch
+---
```

Comment:
I would say that this is a minimal breaking change, maybe it must be minor?

Reply:
:pouce: